pyproject.toml

```diff
@@ -30,7 +30,9 @@ classifiers = [
 ]
 dependencies = [
     # The core package.
-    "signac>=1.8.0",
+    # TODO: Test against the released 2.0.0 once available.
+    #"signac>=2.0.0",
+    "signac @ git+https://github.com/glotzerlab/signac@master",
```

Comment:
Created an issue to track this one https://github.com/glotzerlab/signac-flow/issues/711